### PR TITLE
Add pin title and pin error to pin fragment

### PR DESCRIPTION
**Pull request by Pavlo-Hural**
image

### CODE REVIEW

File: app/src/main/java/com/yterletskyi/happyfriend/features/pin/ui/PinFragment.kt [hunk 1]
```diff
@@ -24,5 +25,9 @@ class PinFragment : BaseBindingFragment<FragmentPinBinding>(
         viewModel.pinProgressLiveData.observe(viewLifecycleOwner) { progress ->
             binding.pinProgressView.step = progress
         }
+
+        with(binding.pinTitle) {
+            text = getString(R.string.pin_enter_title)
```
Comment (yterletskyi) [inline app/src/main/java/com/yterletskyi/happyfriend/features/pin/ui/PinFragment.kt:30]: can we set it via xml with just one line?

File: app/src/main/res/layout/fragment_pin.xml [hunk 2]
```diff
@@ -6,17 +6,44 @@
...
+        app:layout_constraintBottom_toBottomOf="parent"
         app:progressCircleColor="?attr/colorPrimary" />
 
+    <TextView
+        android:id="@+id/pin_error"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:textSize="12sp"
+        android:layout_marginTop="10dp"
+        android:textColor="@color/red"
```
Comment (yterletskyi) [inline app/src/main/res/layout/fragment_pin.xml]: Let's please use `?attr/colorError` for errors. Also, please define it in `themes.xml` and `night/themes.xml` like we have other theme colors for exmaple `colorPrimary` etc.

There is a good article about meterial theme and colors: https://medium.com/over-engineering/setting-up-a-material-components-theme-for-android-fbf7774da739

File: app/src/main/res/values/strings_pin.xml [hunk 3]
```diff
@@ -8,6 +8,7 @@
     <string name="phrase3">Be attentive, there is no way to restore a \n lost PIN so far.</string>
     <string name="action_button_next">Next</string>
     <string name="pin_enter_title">Enter a pin</string>
```
Comment (yterletskyi) [inline app/src/main/res/values/strings_pin.xml]: a bit late but to the party, but please make this string `Enter a PIN` to match the figma design

File: app/src/main/java/com/yterletskyi/happyfriend/features/pin/ui/PinFragment.kt (hunk 1 above)
Comment (Pavlo-Hural) [reply app/src/main/java/com/yterletskyi/happyfriend/features/pin/ui/PinFragment.kt:30]: but if we will want to change text from Enter a PIN to Repeat your PIN, we have to do that from code

Comment (yterletskyi) [reply app/src/main/java/com/yterletskyi/happyfriend/features/pin/ui/PinFragment.kt:30]: oh make sense -- don't change it then

**Review by yterletskyi — APPROVED** [review]

**Pull request merged**